src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,67 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoChanges } from '../../types/TodoChanges';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  someCompletedTodo: boolean,
+  todos: Todo[],
+  setTodos: TodoChanges,
+};
+
+export const TodoList: React.FC<Props> = ({
+  someCompletedTodo,
+  todos,
+  setTodos,
+}) => {
+  const [changeTitle, setChangeTitle] = useState<number | null>(null);
+
+  const handleToggleAll = () => {
+    let allCompleted = todos.filter(todo => todo.completed === false);
```

Comment:
```suggestion
    let allCompleted = todos.filter(todo => !todo.completed);
```

Reply:
not fixed
![image](https://user-images.githubusercontent.com/98764641/232405386-daa78d00-e8e1-48b7-be2b-63ce3dd8f350.png)